.github/workflows/agent_push.yml

```
@@ -0,0 +1,23 @@
+name: agent_push
+on:
+  push:
+    branches: [agent/codex]
+jobs:
+  lint-and-test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)